_conferences/gdg_liverpool_devfest_2016.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG Devfest"
+website: https://docs.google.com/forms/d/e/1FAIpQLSfT8oMoJHyOjNGyDVE15O-TEw5I8ABeVNeJRywj7OXlBHFdqg
```

Comment:
I'd recommend you checking that link in an anonymous session in your Internet browser.